src/components/ImportOnyxState/index.native.tsx

```diff
@@ -13,33 +13,15 @@ import {cleanAndTransformState} from './utils';
 
 const CHUNK_SIZE = 100;
 
-function readFileInChunks(fileUri: string, chunkSize = 1024 * 1024) {
+function readOnyxFile(fileUri: string) {
     const filePath = decodeURIComponent(fileUri.replace('file://', ''));
 
-    return RNFS.exists(filePath)
-        .then((exists) => {
-            if (!exists) {
-                throw new Error('File does not exist');
-            }
-            return RNFS.stat(filePath);
-        })
-        .then((fileStats) => {
-            const fileSize = fileStats.size;
-            let fileContent = '';
-            const promises = [];
-
-            // Chunk the file into smaller parts to avoid memory issues
-            for (let i = 0; i < fileSize; i += chunkSize) {
-                promises.push(RNFS.read(filePath, chunkSize, i, 'utf8').then((chunk) => chunk));
-            }
-
-            // After all chunks have been read, join them together
-            return Promise.all(promises).then((chunks) => {
-                fileContent = chunks.join('');
-
-                return fileContent;
-            });
-        });
+    return ReactNativeBlobUtil.fs.exists(filePath).then((exists) => {
```

Comment:
@TMisiukiewicz, have you considered using [readStream](https://github.com/RonRadtke/react-native-blob-util/wiki/File-System-Access-API#readstreampath-encoding-buffersize-interval-promisernfbreadstream) instead of readFile? 
According to the [docs](https://github.com/RonRadtke/react-native-blob-util/wiki/File-System-Access-API#readfilepath-encodingpromise):
> if the file is large, you should consider use readStream instead.

And since log files could realistically be large, we might hit a limit

Reply:
Yes, this was my initial idea. However, when using it, the app was crashing immediately without any JS error. I found out during debugging with Xcode that this is the error that makes it crash:
```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]'
```

I couldn't find any information how to fix it. I found out that this was reported in the library repository here: https://github.com/RonRadtke/react-native-blob-util/issues/392 but thought the fastest way to unblock me from doing other fixes related to Import is using `readFile`. Works for me with ~15mb files. Maybe it's a good idea to go with this solution first and eventually adjust it once we hit any issues?

Reply:
> Maybe it's a good idea to go with this solution first and eventually adjust it once we hit any issues?

Can confirm, this works for large files on my end too. Agree, let's proceed with the current solution, we can follow-up in case any problems arise